### ISSUE TITLE
Gotta share with your friends if you get a GLEAM light art installation accepted!

### ISSUE THREAD

**Issue by youralien** [promnius/GLEAM#1]
Visited the Albrich gardens a couple weeks ago (from Chicago)! I'd love to return, esp. if it was YOUR awesome digital light art display that I was able to appreciate.

P.S. Sorry for Github lurking! Hope you are well!

**Comment by promnius** [comment]
haha, thanks! It's a 2020 proposal, so it wouldn't be until 2021. I should
have some photos of the product sooner than that though.

**Comment by promnius** [comment]
wait, hold up a second. Are you in chicago now?!?

**Comment by youralien** [comment]
Yep I'm in Chicago! I am doing grad school at Northwestern for Human Computer Interaction stuff! I come up to Wisconsin now a few times a year to go climbing around Madison!

**Comment by promnius** [comment]
Oh snap! I'm down in Chicago every third week or so! We should catch up sometime, maybe after covid becomes less aggressive! And I love climbing, let me know the next time you're in Madison! Where do you climb around here? I thought Devil's Lake was the closest good climbing? I'm sure you're crazy busy with grad school, but if you're ever looking for a project buddy I'd love to make something together! I'm always working on something fun, and always looking for more :) I wandered through some of your github stuff, but I'm not much of a software person anymore, as I'm sure you've noticed ;)

Anyways, I don't seem to have a current phone number for you and a github ticket isn't exactly cutting edge communication, shoot me a text at 608-852-6478 or an email at kylemayersolar@gmail.com, I'd love to catch up and swap stories sometime!
